kernel/network/e1000.c

```diff
@@ -152,6 +177,10 @@ start_e1000(void)
   init_transmit_descriptors();
 }
 
+/*
+ * Fills the next transmit buffer with information on the data to send.
+ * Tells the e1000 that a new packet is available for send and waits until its send.
```

Comment:
until it's send
or
until its sending / transmission

Reply:
`until it's sent` 😛